src/Data/Lens.purs

```diff
@@ -0,0 +1,8 @@
+module Data.Nullable.Lens where
+
+import Data.Lens (prism')
+import Data.Nullable (Nullable, notNull, toMaybe)
+import Data.Profunctor.Choice (class Choice)
+
+_NotNull :: forall p a. (Choice p) => p a a -> p (Nullable a) (Nullable a)
+_NotNull = prism' notNull toMaybe
```

Comment:
What about making this an `Iso` between `Nullable` and `Maybe`. Then you can reuse `_Just`, and you don't need to export `notNull`, which I think has a slightly confusing name.